### ISSUE TITLE
Disallow Order Splitting for reservations

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1300]
_Imported from trac ticket 1300.
Created by **sshah**
Opened in trac: Dec 24, 2011, 11:44:25 AM
Last modified in trac: Jan 9, 2012, 8:46:57 AM_
***
_by sshah:_
What some sales associates are doing is splitting the reservations even before they’re actual orders. They do this to ‘bypass’ rules.

We should only allow splitting an order once they are actual sales orders.

**Comment by kbower** [comment]
_**sshah** trac comment on Dec 24, 2011, 11:57:36 AM:_

(Changeset [[2082]](../commit/fb199edb0e405f5dadc705f9e2c0c2436a0a760d))
Fixes #1300 Added new rule to disallow splitting of res orders

**Comment by kbower** [comment]
_**sshah** trac status change on Dec 24, 2011, 11:57:36 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 6, 2012, 6:50:32 PM:_

Yeah, the split isn't a RES, the split is a SAL.  We'll have to make the rule a little cleverer.

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 6, 2012, 6:50:32 PM:_ **reopened**

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 6, 2012, 6:50:44 PM:_ **new**

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 6, 2012, 6:58:55 PM:_

spoiler:

delete the Order rule, create a new !CustomerOrder rule:

```python
if len(customerorder.orders) > 1:
    for o in customerorder.orders):
        if o.type == u'RES':
            o.error('Splitting reservations is not allowed')
            break
```

**Comment by kbower** [comment]
_**mpisarski** trac comment on Jan 9, 2012, 8:46:57 AM:_

(Changeset [[2124]](../commit/49af3d515df8913b510b2c78d3938a30426c0819))
 * fixes #1300 - Disallow Order Splitting for reservations on save.

**Comment by kbower** [comment]
_**mpisarski** trac status change on Jan 9, 2012, 8:46:57 AM:_ **closed**
